### ISSUE TITLE
Support multiple plugin instances for a single community

### ISSUE THREAD

**Issue by gigxz** [metagov/gateway#50]
Support having multiple plugin instances of the same type for a single community. For example community X should be able to have 2 enabled instance of the `RevShare` Plugin, each with different configurations.

**Comment by mashton** [comment]
Once we have support for this, we may want to update the Loomio plugin. Currently, a community that uses Loomio subgroups would use one Loomio Plugin instance, and with all the API keys for each subgroup uploaded into the Plugin config. It may be better for each Loomio subgroup to have its own Loomio Plugin instance, so they can be added and removed independently. @shaunagm

**Comment by shaunagm** [comment]
@mashton yeah that makes sense. 

Supporting this would involve making use of community_platform_id, right? We want to make it possible for a community to have multiple slack plugins, for instance, but only one of each specific slack?

**Comment by mashton** [comment]
Yes exactly. community_platform_id will certainly make this easier to implement.

**Comment by thelastjosh** [comment]
Was this issue actually resolved?
